api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,35 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the oathkeeper v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=oathkeeper.ory.sh
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "oathkeeper.ory.sh", Version: "v1alpha1"}
```

Comment:
Should `version` be the version of this controller or the software version?

Reply:
This is the CRD Api version. It follows general K8s [conventions](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning).
At the moment alpha version seems to be a reasonable choice.